### ISSUE TITLE
WCAG issue (1.1.1 Non-text Content)

### ISSUE THREAD

**Issue by sorensensig** [digdir/baksia#917]
List of issues related to 1.1.1 Non-text Content found during WCAG walkthrough of Baksia:

```[tasklist]
### General issues
- [x] Text on [the third image](https://github.com/user-attachments/assets/43360bee-3687-414f-a835-9fd57c5d87e2) under the [legal principles page](https://baksia.digdir.no/produktutviklingsmodell/prinsipper/juridisk/) is not readable.
- [x] Redundant alternative text on all profile images on each team page under [teams](https://baksia.digdir.no/teams/)
```

```[tasklist]
### Incorrect alternative text
- [x] Alternative text does not match what the illustration is showing on all [principle pages](https://baksia.digdir.no/produktutviklingsmodell/prinsipper/)
- [x] Alternative text does not match what the illustration is showing on all team pages [teams page](https://baksia.digdir.no/teams/).
- [x] Alternative text does not match what the illustration is showing on the [OKR page](https://baksia.digdir.no/produktutviklingsmodell/okr/)
- [x] Alternative text does not match what the illustrations are showing on the [section page](https://baksia.digdir.no/brukeropplevelse-og-datadeling/seksjoner/)
```

```[tasklist]
### Missing alternative text
- [x] Missing alternative text on team typologies illustration on the [teams page](https://baksia.digdir.no/teams/)
- [x] Missing alternative text on all images on the [scrum/agile page](https://baksia.digdir.no/produktutviklingsmodell/metodikk/scrum/)
- [x] Missing alternative text on the top image of the [design thinking page](https://baksia.digdir.no/produktutviklingsmodell/metodikk/design-thinking/)
- [x] Missing alternative texts on all images under the [DevOps page](https://baksia.digdir.no/produktutviklingsmodell/metodikk/devops/)
- [x] Missing alternative text on head illustration on the [product group page](https://baksia.digdir.no/produktgrupper/) and all sub-pages
- [x] Missing alternative text on header illustration on the [internal reporting page](https://baksia.digdir.no/brukeropplevelse-og-datadeling/rapportering/)
- [x] Missing alternative text on header illustration on the [onboarding page](https://baksia.digdir.no/brukeropplevelse-og-datadeling/onboarding/)
- [x] Missing alternative text on header illustration on the [tools page](https://baksia.digdir.no/brukeropplevelse-og-datadeling/verktoy-og-infokanal/)
```
